config/locales/server.en.yml

```diff
@@ -25,3 +26,15 @@ en:
     tech_support:
       name: "Tech Support"
       description: "10 Accepted answers"
+
+  education:
+    topic_is_solved: |
+      ### This topic has been solved
+
+      Only reply here if:
+
+      - You have additional details
+
+      - The solution doesn't work for you
+
+      If you have an unrelated issue, please [start a new topic](%{base_url}) instead.
```

Comment:
Do we want to use `/new-topic` to open the composer for the user?

Reply:
Yeah agree that would be a lot better